theano/contrib/mkl/basic_ops.py

```diff
@@ -0,0 +1,472 @@
+import theano.tensor as T
+from theano.gof import Apply, Op
+from theano.tensor.blas import ldflags
+from theano.tensor.nnet.abstract_conv import get_conv_output_shape
+from theano.contrib.mkl.mkl_helper import header_text
+from theano.contrib.mkl.mkl_type import MKLNdarrayType
+
+
+class MKLOp(Op):
+    def c_lib_dirs(self):
+        return ldflags(libs=False, libs_dir=True)
+
+    def c_libraries(self):
+        return ldflags()
+
+    def c_compile_args(self):
+        compile_args = ldflags(libs=False, flags=True)
+        compile_args += super(MKLOp, self).c_compile_args()
```

Comment:
This line will cause an error to be raised an that will cause this method to be ignored. I think you should just delete this line.